acceptance_tests/cypress/integration/lotse_flow.spec.js

```diff
@@ -121,7 +121,7 @@ context('Acceptance tests', () => {
             cy.get('#familienstand-1').check()
             cy.get('#familienstand_date').clear().type(older_date)
             cy.get('label[for=familienstand_married_lived_separated-no]').click()
-            cy.get('label[for=familienstand_confirm_zusammenveranlagung]').first().click()
+            cy.get('label[for=familienstand_confirm_zusammenveranlagung]').should('be.visible')
```

Comment:
Why do we not click on it anymore?

Reply:
The click didn't do anything as it is overwritten right away with the next click. What I really did by clicking it is to check that you can click it. So that it's visible. Now it's more clear.